Sources/KlaviyoSwift/InternalAPIModels.swift

```diff
@@ -30,11 +31,11 @@ extension KlaviyoAPI.KlaviyoRequest {
                     let properties: AnyCodable
                     enum CodingKeys: String, CodingKey {
                         case email
-                        case phoneNumber
-                        case externalId
-                        case anonymousId
-                        case firstName
-                        case lastName
+                        case phoneNumber = "phone_number"
```

Comment:
I ran into some encoding issues with this request and codeable and had to fix it up.